rule_schema_v1.yaml

```diff
@@ -212,6 +215,7 @@ $defs:
           - required: [ pattern-either ]
           - required: [ pattern-not ]
           - required: [ pattern-inside ]
+          - required: [ anywhere ]
```

Comment:
I guess pattern-anywhere here? to be consistent with the other?

Reply:
I don't feel strongly; if anything I would rather just make it `semgrep-internal-pattern-anywhere` and then just remove that prefix & document this when we look to transition to the "new" syntax at some future point.

More broadly, consistency-wise, it is already inconsistent with how `inside`/`pattern-inside` works in the old syntax since it takes any pattern rather than just a leaf (whereas `pattern-inside` only lets you do a leaf pattern, in the sense of a string rather than a formula)

Reply:
True, I forgot about that.

Reply:
go for semgrep-internal-pattern-anwhere in old syntax and just 'anywhere' in new syntax, to give another incentive to use the new syntax :)